.github/workflows/merge.yml

```diff
@@ -21,4 +21,6 @@ jobs:
       
       - name: docker build
         run: |
-          make build-docker gcr-repo=${{ secrets.RELEASE_REPO }}
+          make build-docker-ig tag=${{ github.sha }}
+          docker tag eu.gcr.io/${{ secrets.DEV_REPO}}:{{ github.sha }} eu.gcr.io/${{ secrets.RELEASE_REPO}}:latest
+          docker push eu.gcr.io/${{ secrets.RELEASE_REPO}}:latest
```

Comment:
Must be the same repository to have the same image tagged with latest and github.sha.
- docker push eu.gcr.io/${REPO}:latest
- docker push eu.gcr.io/${REPO}:{{ github.sha }}